src/components/controls/Button.tsx

```diff
@@ -7,13 +7,14 @@ interface IButtonProps {
     type: ButtonType;
     text: string;
     className?: string;
+    disabled?: boolean;
```

Comment:
does this need to be optional if defaultProps gives it a value?

Reply:
yes